File: Java/Lista 3/L03EX02/src/Premium.java

public class Premium extends Conta{

	public Premium(String nome, String nun_conta, Double saldo) {
		super(nome, nun_conta, saldo);
	}
	
	
	public Double getTarPremium() {
		return (this.getSaldo() * 0.02);
	}
	
    public void mostraTarPremium() {
    	System.out.println("Tarifa: R$ " + this.getTarPremium());
    }
    
    
    public void mostraPremium() {
    	System.out.println("Numero da Conta........: " + this.getNun_conta());
    	System.out.println("Nome...................: " + this.getNome());
    	System.out.println("Saldo..................: R$ " + this.getSaldo());
    	System.out.println("Tarifa Premium (2%)....: R$ " + this.getTarPremium());
    }
	
}

File: Swift/L3/L03EX02/L03EX02/Premium.swift
// Premium.swift - L03EX02

import Foundation

public class Premium: Conta{
    
  
    public func getTarPremium() -> Double{
      return (self.getSaldo() * 0.02)
    }
  
    public func printTarPremium(){
        print("A tarifa Premuim (2%) do(a) \(self.getNome()): R$ \(self.getTarPremium())\n")
    }
    
    public func mostraClientePremium(){
         print("Conta Número: \(self.getNumConta())")
         print("Nome Cliente: \(self.getNome())")
         print("saldo: R$ \(self.getSaldo())")
         print("A tarifa Premuim (2%): R$ \(self.getTarPremium())\n")
    }
    
}



		
